docs/docusaurus/docs/guides/connecting_to_your_data/fluent/batch_requests/how_to_request_data_from_a_data_asset.md

```diff
@@ -59,6 +59,32 @@ For `dataframe` Data Assets, the `dataframe` is always specified as the argument
 ```python name="tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/get_existing_data_asset_from_existing_datasource_pandas_filesystem_example.py build_batch_request_with_dataframe"
 ```
 
+## Extract a Batch from a Batch Request (Optional)
+
+You can use the Python slice function to remove a subset of data from a Batch Request and use a specific selection of records to build Metrics, Validations, and Profiles. In the following example, data is sliced and filtered by column, but you can also use other parameters such as time or date to slice and filter data.
+
+1. Run the following code to retrieve an entire table of data from a SQL datasource:
+
+    ```python
+    table_asset = datasource.add_table_asset(name=asset_name, table_name=asset_table_name)
+    ```
```

Comment:
Anything in a code block should be pulled from a snippet in an example script that is under test, to ensure that it doesn't break if engineering makes changes to the API.